### DISCUSSION TITLE
FBX skeletally animated models are loading incorrectly after migration from Assimp 3.1.1 to 5.2

### DISCUSSION THREAD

**Discussion by prunoid** [assimp/assimp#4561] [Q&A] [answered]
Blender 3.0 FBX exported skeletal animated models previously loaded correctly in Assimp 3.1.1 but they no longer load correctly in 5.2

I've attached screen shots for the 3.1.1 FBX bin load and the 5.2 load.
Before I start trawling through the FBX importer code, can anyone help out?
FYI:  The FBX file was generated with Blender 3.0.
Thanks.

Assimp 3.1.1:
![image](https://user-images.githubusercontent.com/15125149/172027226-807784a9-11fc-4156-87af-62e608b39f07.png)

Assimp 5.2:
![image](https://user-images.githubusercontent.com/15125149/172027241-ca58f7ab-fdd7-48f8-b23f-ebe1ab9c33e3.png)

**Comment by kimkulling** [comment, marked as answer]
Upps, have you tried 5.4 instead? We made a lot of changes in this area.

**Reply by prunoid** [reply]
Thanks @kimkulling , the response is much appreciated.

5.2.3 bug fix works fine, so the issue is in 5.2.4 bug fix.
Looking at the updates from 5.2.3 to 5.2.4, the problem is in the following update:

FbxConverter: update bone mOffsetMatrix by @Mykhailo418 in https://github.com/assimp/assimp/pull/4490

I'll stick with 5.2.3 for now. 

Cheers Will

**Reply by omichel** [reply]
I jump bumped into the same problem: https://github.com/assimp/assimp/issues/4574
